src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -898,6 +960,29 @@ public String toCommandLineString(){
             }
         }
 
+        /**
+         * @param argumentDefinitionMap a map from argument names to {@link ArgumentDefinition}
+         * @return the set of arguments that this argument is mutually exclusive with (including itself in the set)
+         */
+        public Set<ArgumentDefinition> getMutuallyExclusiveArguments(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            final Set<ArgumentDefinition> mutexArguments = mutuallyExclusive.stream().map(argumentDefinitionMap::get).collect(Collectors.toSet());
+            mutexArguments.add(this);
+            return mutexArguments;
+        }
+
+        public String composeMissingArgumentMessage(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            if(mutuallyExclusive.isEmpty()){
+                return String.format("required argument --%s was not specified", getLongName());
```

Comment:
Right now we output the individual element type as the type of the input, but for collections we could change so that it mentions it's a collection

current `--Input, -I: String` 

proposed
1.  `--Input, -I: List<String>` most accurate
2.  `--Input, -I: String [String ... ]` posixy?
3.  `--Input, -i: String[]`  implies collection with minimal characters, but isn't strictly accurate.